src/styles/calendar.scss

```
@@ -0,0 +1,7 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: #{$gap};
+  padding: 10px;
+  max-width: #{$calendar-width};
+}
```

Don't forget about the scss nesting
```suggestion
  max-width: #{$calendar-width};
  
  &__day { ...
}
```